filebeat/beater/filebeat.go

```diff
@@ -75,7 +75,10 @@ func New(b *beat.Beat, rawConfig *common.Config) (beat.Beater, error) {
 func (fb *Filebeat) modulesSetup(b *beat.Beat) error {
 	esConfig := b.Config.Output["elasticsearch"]
 	if esConfig == nil || !esConfig.Enabled() {
-		return fmt.Errorf("Filebeat modules configured but the Elasticsearch output is not configured/enabled")
```

Comment:
An alternate version (in case you want to shift the focus away from the idea that ES is required): 

Filebeat is unable to load the Ingest Node pipelines for the configured modules because the Elasticsearch output is not configured/enabled. If you have already loaded the Ingest Node pipelines or are using Logstash pipelines, you can ignore this warning.